R/build_lookup.R

```diff
@@ -139,15 +167,13 @@ build_lookup_from_fastas <- function(data_dir, species) {
 
   lookup <- extract_imgt_genes(data_dir)
 
-  # Create the 10X column by removing allele info (e.g. *01) and slash from "/DV".
-  # Do this by substituting "DV" for "/DV" in what's left of the TCR gene name
-  # after removing the last three characters
+  # Create the 10X column
   lookup[["tenx"]] <- sub("/DV", "DV", substr(
     lookup[["imgt"]], 1,
     nchar(lookup[["imgt"]]) - 3
   ))
 
-  # Create Adaptive columns by adding letters, 0's, removing /DV and renaming /OR
+  # Create Adaptive columns
   adaptive_replacements <- c(
```

Comment:
Not to change now unless you want to, probably open an issue for later: 

This vector seems like something that doesn't need to live in the function, and instead put outside of it and just call it from within the function.

Reply:
Easy to do, fixed in [8db5c0d](https://github.com/seshadrilab/tcrconvertr/pull/14/commits/8db5c0d5bbec7fee0fc7189275c73377917eac0c)